packages/sui-bundler/README.md

```diff
@@ -54,6 +56,29 @@ $ sui-bundler dev
 
 While developing your app, you will have HMR (Hot Module Reloading). Default port for your website is 3000, but it will assign automatically a free port for you if this one is busy.
 
+#### How to link packages
+
+`sui-bundler` give us the oportunity to link `npm` packages with `--link-package`. This argument accepts relative and absolute paths.
```

Comment:
Maybe you could add why we recommend to link the packages this way and not with npm link

Reply:
I know that one of the main reasons I'm using it. With `npm unlink` you have to reinstall it because it removes it, --link-package doesn't remove anything